gdsctools/regression.py

```diff
@@ -998,7 +998,7 @@ def barplot(df, colname, color="sign",colors=("b", "r"),
     df.loc[df['sign'] == True, 'sign'] = 'r'
     df.loc[df['sign'] == False, 'sign'] = 'b'
 
-    colors = "".join(df['sign'])
+    colors = list(df['sign'])
```

Comment:
This was not an explicit deprecation warning, but matplotlib didn't seem to like a list of color code letters. This change updates the `colors`-variable from a string like `'rrbbr'` to a list `['r', 'r', 'b', 'b', 'r']`, which seems to work fine with both old and new versions of matplotlib

Reply:
I had seen this one but with no solution so glad to see this neat solution.